rosdep/python.yaml

```diff
@@ -689,6 +689,18 @@ python-evdev:
     vivid: [python-evdev]
     wily: [python-evdev]
     xenial: [python-evdev]
+python-expiringdict:
+  debian:
+    jessie: [python-expiringdict]
+    sid: [python-expiringdict]
```

Comment:
Please don't add rules for sid explicitly, we don't target it because it's a rolling target. 

And since these are all the same it doesn't need to have each version called out individually.